examples/verifying_fd_afd.py

```diff
@@ -0,0 +1,93 @@
+import desbordante
+import pandas as pd
+
+
+GREEN_CODE = '\033[1;42m'
+RED_CODE = '\033[1;41m'
+BLUE_CODE = '\033[1;46m'
+DEFAULT_COLOR_CODE = '\033[1;49m'
+DATA_FOR_FD = pd.read_csv('examples/datasets/duplicates_short.csv', header=[0])
+DATA_FOR_AFD = pd.read_csv('examples/datasets/DnD.csv', header=[0])
```

Comment:
It'd be preferable if these variables were function parameters instead of globals.